appium/tests/helpers/TouchHelper.ts

```diff
@@ -29,6 +30,27 @@ class TouchHelper {
       {action: 'release', options: {}},
     ]);
   }
+
+  static scrollDownToElement = async (element: WebdriverIO.Element) => {
+    const { width, height } = await driver.getWindowSize();
+    const anchor = width / 2;
+    const startPoint = height * 0.25;
+    const endPoint = height * 0.15;
```

Comment:
could probably do a more "generous" scroll like from 0.75 to 0.15. So that we don't have to scroll so many times.